### PR TITLE
V1.10.0

### PR DESCRIPTION
**Pull request by lina1101**

### CODE REVIEW

File: sokannonser/repository/platsannonser.py [hunk 1]
```diff
@@ -120,10 +137,19 @@ def find_platsannonser(args, querybuilder, start_time=0, x_fields=None):
     return transform_platsannons_query_result(args, query_result, querybuilder)
 
 
-def suggest(args, querybuilder, start_time=0, x_fields=None):
+def complete_suggest(args, querybuilder, start_time=0, x_fields=None):
```
Comment (halge) [inline sokannonser/repository/platsannonser.py]: @lina1101 x-fields not used.

File: sokannonser/repository/platsannonser.py [hunk 2]
```diff
@@ -120,10 +137,19 @@ def find_platsannonser(args, querybuilder, start_time=0, x_fields=None):
...
     if start_time == 0:
         start_time = int(time.time() * 1000)
-    query_dsl = querybuilder.create_suggester(args)
+
+    input = args.get(settings.TYPEAHEAD_QUERY)
```
Comment (halge) [inline sokannonser/repository/platsannonser.py]: @lina1101 input is built-in name.

File: sokannonser/repository/platsannonser.py [hunk 3]
```diff
@@ -140,26 +166,90 @@ def suggest(args, querybuilder, start_time=0, x_fields=None):
...
+                        'found_phrase': value,
                         'type': key.split('-')[0],
-                        'occurrences': None
+                        'occurrences': 0
+                    }
+                )
+
+    # check occurrences even i think it will take some trouble and stupid
+    query_result['aggs'] = suggest_check_occurence(aggs[:50], args, querybuilder)
+    log.debug(query_result['aggs'])
```
Comment (halge) [inline sokannonser/repository/platsannonser.py]: @lina1101 add some phrase with log.

File: sokannonser/repository/platsannonser.py (hunk 1 above)
Comment (lina1101) [reply sokannonser/repository/platsannonser.py]: I know this one , just keep it for right now. I will change it future

File: sokannonser/repository/platsannonser.py [hunk 4]
```diff
@@ -140,26 +166,90 @@ def suggest(args, querybuilder, start_time=0, x_fields=None):
...
+    if start_time == 0:
+        start_time = int(time.time() * 1000)
+
+    input = args.get(settings.TYPEAHEAD_QUERY)
+    query_dsl = querybuilder.create_phrase_suggester(input, args)
+    log.debug("Query constructed after %d milliseconds."
+              % (int(time.time() * 1000) - start_time))
+    try:
+        log.debug("ARGS %s => QUERY: %s" % (args, json.dumps(query_dsl)))
+        log.debug(query_dsl)
```
Comment (halge) [inline sokannonser/repository/platsannonser.py]: @lina1101 double log?

File: sokannonser/repository/platsannonser.py [hunk 5]
```diff
@@ -140,26 +166,90 @@ def suggest(args, querybuilder, start_time=0, x_fields=None):
...
+        query_result = elastic.search(index=settings.ES_INDEX, body=query_dsl)
+        log.debug(query_result)
```
Comment (halge) [inline sokannonser/repository/platsannonser.py]: @lina1101 add some phrase.

File: sokannonser/repository/platsannonser.py [hunk 6]
```diff
@@ -140,26 +166,90 @@ def suggest(args, querybuilder, start_time=0, x_fields=None):
...
+        log.debug("Elastic results after %d milliseconds."
+                  % (int(time.time() * 1000) - start_time))
+    except exceptions.ConnectionError as e:
```
Comment (halge) [inline sokannonser/repository/platsannonser.py]: @lina1101 use log.error

File: sokannonser/repository/platsannonser.py [hunk 7]
```diff
@@ -140,26 +166,90 @@ def suggest(args, querybuilder, start_time=0, x_fields=None):
...
+        logging.exception('Failed to connect to elasticsearch: %s' % str(e))
```
Comment (halge) [inline sokannonser/repository/platsannonser.py]: @lina1101 use log.error

File: sokannonser/repository/platsannonser.py (hunk 2 above)
Comment (lina1101) [reply sokannonser/repository/platsannonser.py]: changed

File: sokannonser/repository/platsannonser.py (hunk 3 above)
Comment (halge) [reply sokannonser/repository/platsannonser.py]: ok later

File: sokannonser/repository/platsannonser.py (hunk 5 above)
Comment (halge) [reply sokannonser/repository/platsannonser.py]: later

**Pull request merged**